server/server.go

```diff
@@ -559,6 +559,9 @@ func (s *Server) Run() error {
 	if err := s.startEtcd(s.ctx); err != nil {
 		return err
 	}
+	failpoint.Inject("delayStartServer", func() {
+		time.Sleep(2 * time.Second)
```

Comment:
Is it possible to inject an input channel and let it wait for an input after mcs.NewTestTSOCluster(ctx, 2, addr) is completed? This way could make the test more deterministic and stable.